code/datums/datumvars.dm

```diff
@@ -63,6 +63,25 @@
 		boutput(usr, "<span class='alert'>Could not find [S] in the Global Variables list!!</span>" )
 		return
 
+/client/proc/debug_ref_variables()
+	SET_ADMIN_CAT(ADMIN_CAT_DEBUG)
+	set name = "View Ref Variables"
+
+	if (src.holder.level < LEVEL_ADMIN)
+		alert("You must be at least an Administrator to use this command.")
+		src.audit(AUDIT_ACCESS_DENIED, "tried to call debug_ref_variables while being below Administrator rank.")
+		return
+
+	var/ref = input(src, "Enter a ref to view the variables of", "Debug Ref Variables")
```

Comment:
replace with a verb argument imo so you can use it from the command bar

Reply:
added